pkg/agent/bakerapi.go

```diff
@@ -174,3 +175,11 @@ func findSIGImageConfig(sigConfig datamodel.SIGAzureEnvironmentSpecConfig, distr
 
 	return nil
 }
+
+func (agentBaker *agentBakerImpl) GetCachedVersionsOnVHD() (map[string]datamodel.ProcessedManifest, map[string]datamodel.ContainerImage,
```

Comment:
consider having this return a `CachedOnVHD` instead?

Reply:
was going to say the same, seems cleaner?